mineinabyss-features/src/main/kotlin/com/mineinabyss/features/okibotravel/OkiboTravelFeature.kt

```diff
@@ -1,32 +1,35 @@
 package com.mineinabyss.features.okibotravel
 
+import com.mineinabyss.features.helpers.di.Features
+import com.mineinabyss.features.helpers.di.Features.okiboLine
 import com.mineinabyss.idofront.commands.arguments.optionArg
 import com.mineinabyss.idofront.commands.extensions.actions.playerAction
 import com.mineinabyss.idofront.config.config
 import com.mineinabyss.idofront.di.DI
 import com.mineinabyss.idofront.messaging.error
+import com.mineinabyss.idofront.messaging.logSuccess
 import com.mineinabyss.idofront.messaging.success
 import com.mineinabyss.idofront.plugin.listeners
 import com.mineinabyss.mineinabyss.core.AbyssFeature
+import com.mineinabyss.mineinabyss.core.Configurable
 import com.mineinabyss.mineinabyss.core.MineInAbyssPlugin
 import com.mineinabyss.mineinabyss.core.abyss
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
 
 @Serializable
 @SerialName("okibotravel")
-class OkiboTravelFeature : AbyssFeature {
-    private fun setupOkiboContext() {
-        DI.remove<OkiboLineContext>()
-        DI.add<OkiboLineContext>(object : OkiboLineContext {
-            override val config: OkiboTravelConfig by config("okiboTravel") { abyss.plugin.fromPluginPath(loadDefault = true) }
-        })
-    }
+class OkiboTravelFeature : AbyssFeature, Configurable<OkiboTravelConfig> {
+    @Transient
+    override val configManager = config("okiboTravel", abyss.dataPath, OkiboTravelConfig(), onReload = {
+        spawnOkiboMaps()
+        logSuccess("Okibo-Context reloaded")
+    })
+
+    override val dependsOn = setOf("Train_Carts", "TCCoasters")
```

Comment:
this onReload, what is the logic behind that? is that to do with the configReload command? or is there a like /mineinabyss reload FEATURE that triggers this?

Reply:
Will add that in the future, but it's for config reload specifically